bindings_ffi/src/mls.rs

```diff
@@ -659,6 +668,32 @@ impl FfiGroup {
         Ok(group_name)
     }
 
+    pub async fn update_group_image_url(&self, group_image_url: String) -> Result<(), GenericError> {
+        let group = MlsGroup::new(
+            self.inner_client.context().clone(),
+            self.group_id.clone(),
+            self.created_at_ns,
+        );
+
+        group
+            .update_group_image_url(&self.inner_client, group_image_url)
+            .await?;
+
+        Ok(())
+    }
+
+    pub fn group_image_url(&self) -> Result<String, GenericError> {
```

Comment:
Can we rename all this to `group_image_url_square` or something else more specific? It's going to be difficult to rename later since the field is in the metadata and group permissions, and I can totally imagine people wanting groups with multiple sizes and shapes of image (a wide header for example).